tools/legacy/BuildMonitor/Legacy_BuildMonitor.csproj

```diff
@@ -38,6 +38,7 @@
     <IncludeDebugSymbolsInLocalVSIXDeployment>true</IncludeDebugSymbolsInLocalVSIXDeployment>
     <CopyBuildOutputToOutputDirectory>true</CopyBuildOutputToOutputDirectory>
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+    <DeployExtension>False</DeployExtension>
```

Comment:
never mind, found what happened here.


---
In reply to: [107792278](https://github.com/Microsoft/WinObjC/pull/2311#discussion_r107792278) [](ancestors = 107792278)